vendor/vendor.json

```diff
@@ -441,28 +441,28 @@
 			"revisionTime": "2017-04-20T09:04:30Z"
 		},
 		{
-			"checksumSHA1": "vrp0KSG3kZimXujiJMc/NTf2w3M=",
+			"checksumSHA1": "+Yj9sWqspkwTTYsSpnDACT7ZPU0=",
```

Comment:
Why master it would be trickier? 3d party dependency gets new release and once we decide to update to it we do it first in master.